### PR TITLE
Integrating Cocktail API into handling the app_mention event

### PR DESCRIPTION
**Pull request by HidemichiShimura**
# Description

- This integration allows my chat bot to post a random drink name when it is mentioned instead of saying "Hi"

# Implementation

- Created a function lookupRandomCocktail which returns a random drink name fetched from Cocktail API
- Pass a random drink name fetched from Cocktail API to the function chatPostMessage for its argument

### CODE REVIEW

File: src/functions/lookupRandomCocktail.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+function lookupRandomCocktail (){
+    const options = {
+        "port": null,
+        "headers": {
+            "X-RapidAPI-Key": process.env.X_RAPIDAPI_KEY,
+            "X-RapidAPI-Host": "the-cocktail-db.p.rapidapi.com",
+            "useQueryString": true
+        }
+    };
```
Comment (ryonryon) [inline src/functions/lookupRandomCocktail.js:14]: options seems constant, you can take it out from the function

File: src/functions/lookupRandomCocktail.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const dotenv = require("dotenv");
```
Comment (ryonryon) [inline src/functions/lookupRandomCocktail.js:1]: This file name looks good. very self-descriptive.

File: src/functions/lookupRandomCocktail.js (hunk 1 above)
Comment (HidemichiShimura) [reply src/functions/lookupRandomCocktail.js:14]: I modified the code and created another pull request #40
Could you check the code modification in the new pull request?

File: src/functions/lookupRandomCocktail.js (hunk 2 above)
Comment (HidemichiShimura) [reply src/functions/lookupRandomCocktail.js:1]: Thank you!

**Pull request merged**